.circleci/config.yml

```diff
@@ -0,0 +1,215 @@
+      - image: circleci/android:api-27-alpha
+    environment:
+      JVM_OPTS: -Xmx3200m     
+    steps:
+      - checkout   
+
+      - run:
+          name: set version
+          command: |
+            set -x
+            export circleci_tag=${CIRCLE_TAG}
+            export version=$(echo "$circleci_tag" | sed 's|.*v\([0-9\.]*\).*|\1|')
+            echo "version:$version"
+            sed  -i   "s/@SDK_VERSION@/$version/"  gradle.properties  
+            cat gradle.properties 
+      - run:
+          name: generate documents
+          command: |
+            bash gradlew generateJavadocs
+            git checkout -- gradle.properties 
+
+      - run:
+          name: copy documents
+          command: |
+            git config --local user.name "${CIRCLE_PROJECT_USERNAME}"
+            git checkout  gh-pages
+            cp -R build/docs/javadoc/*   docs/reference/
+
+      - run:
+          name: checkin documents
+          command: |
+            git add docs/reference
+            git commit -m "Update document via CircleCI"
+            git push -q https://${GH_TOKEN}@github.com/aws-amplify/aws-sdk-android.git  
+
+  release_maven:
+    working_directory: ~/code
+    docker:
+      - image: circleci/android:api-27-alpha
+    environment:
+      JVM_OPTS: -Xmx3200m
+    resource_class: large
+ 
+    steps:
+      - checkout
+      - run:
+          name: install aws cli
+          command: |
+            sudo pip install awscli
+      - run:
+          name: install android build tools
+          command: |
+            echo "${HOME}"
+            echo "$JAVA_HOME"
+            echo "$ANDROID_HOME"
+            echo "$PATH"          
+            ls  "$ANDROID_HOME"
+            ls  "$ANDROID_HOME/platforms/android-27"            
+            echo 'export ANDROID_PLATFORM="27"' >> $BASH_ENV
+            echo 'export ANDROID_BUILDTOOL_VERSION="27.0.1"' >> $BASH_ENV
+
+            echo "build-tools;$ANDROID_BUILDTOOL_VERSION" 
+
+            sdkmanager "platforms;android-21" "build-tools;27.0.1"  "extras;google;m2repository" "extras;android;m2repository"          
+                
+      - run:
+          name: download gpghome
+          command: |
+            aws s3 cp s3://gpghome-csun/gpghome.zip gpghome.zip
+            unzip -a gpghome.zip
+            echo "gpghome contents:"
+            ls gpghome
+
+      - run:
+         name: install maven
+         command: |
+          sudo apt-get install maven
+      - run:
+          name: Download Dependencies
+          command: bash gradlew androidDependencies
+
+      - run:
+          name: publish to maven
+          command: |
+            echo "${HOME}"
+            echo "$JAVA_HOME"
+            echo "$ANDROID_HOME"
+            echo "$M2_HOME"
+            bash CircleciScripts/maven-release.sh
+
+workflows:
+  version: 2
+  build_and_test:
+    jobs: 
+      - build:
+          filters:
+            branches:
+              ignore: /gh-.*/
+  #     - unittest:
```

Comment:
Can we remove the commented config?

Reply:
We want to disable all jobs except to build currently